pkg_api/server/nl_processing.py

```diff
@@ -15,6 +33,43 @@ def post(self) -> Tuple[Dict[str, str], int]:
         Returns:
             A tuple with a dictionary containing a message, and the status code.
         """
-        # TODO: Implement this method following the approved pipeline.
-        # See issue: https://github.com/iai-group/pkg-api/issues/78
-        return {"message": "Not implemented."}, 400
+        data = request.json
```

Comment:
By checking the fields provided we can give a specific and informative error message in case a field to open the PKG or perform the query is missing.